v2/terraform_flannel.py

```diff
@@ -7,6 +7,9 @@
 import shutil
 import yaml
 import json
+import log
+
+logging = log.getLogger(__name__)
```

Comment:
Log object is set up in the base class here: https://github.com/e2e-win/k8s-ovn-ovs/blob/master/v2/ci.py#L21

Why do we need it here?